coderd/httpmw/apikey.go

```diff
@@ -238,6 +238,17 @@ func ExtractAPIKey(rw http.ResponseWriter, r *http.Request, cfg ExtractAPIKeyCon
 		// Tracks if the API key has properties updated
 		changed = false
 	)
+
+	if key.ExpiresAt.Before(now) {
+		return optionalWrite(http.StatusUnauthorized, codersdk.Response{
+			Message: SignedOutErrorMessage,
+			Detail:  fmt.Sprintf("API key expired at %q.", key.ExpiresAt.String()),
+		})
+	}
```

Comment:
Suggestion: Extract `now` assignment and move above the `var` declaration to relevant code-paths closer to each other.

Reply:
Make sense to move this up